src/main/java/bit/javaoop/Employee.java

```diff
@@ -7,6 +7,12 @@ public Employee(String name, double salary) {
         super(name, salary);
     }
 
+    public void answerQuestions() {
+        System.out.println("What is your name?\n" + "My name is: " + this.getName() + "\n");
+        System.out.println("What is your salary?\n" + "I earn: " + this.getSalary() + "\n");
+        System.out.println("Are you satisfied?\n" + this.isSatisfied() + "\n");
+    }
+
     @Override
     public boolean isSatisfied() {
         return getSalary() > 10_000;
```

Comment:
Takie magiczne liczby jak to 10_000 można wyekstrahować do stałej(ctrl + alt + C) i jakoś nazwać np. SATISFYING_SALARY dużo lepiej się to wtedy czyta a gdybyśmy wykorzystywali tą liczbę jeszcze gdzieś w kodzie i zdecydowalibyśmy się potem na zmianę jej wystarczyłoby zmienić wartość stałej.